styles/_base.scss

```diff
@@ -181,4 +181,9 @@ ul {
 
 .section-content {
   width: 100%;
+}
+
+.nabucasa-zigbee-flasher {
+  --mdc-theme-primary: var(--accent-color);
```

Comment:
Im not sure which color would be best here `$brand_color` probably.
`--mdc-theme-primary: $brand_color;`